### PR TITLE
add graphs placeholders

### PR DESCRIPTION
**Pull request by pvelesko**

### CODE REVIEW

File: samples/CMakeLists.txt [hunk 1]
```diff
@@ -91,12 +91,12 @@ set(SAMPLES
...
     11_device
     hipStreamSemantics
-    hipMultiThreadAddCallback
+    #hipMultiThreadAddCallback
     hipInfo
     hipSymbol
     hip-cuda
     printf
-    texture
+    #texture
```
Comment (linehill) [inline samples/CMakeLists.txt:99]: This line was commented out in the `add graphs placeholders` patch. Was this intentional?

Comment (pvelesko) [reply samples/CMakeLists.txt:99]: Yes. It looked to me that these samples are the same as the unit tests since at least one of them had the same name.

**Comment by pvelesko** [comment]
find ./ -name "hipTextureObjFetchVector.cc"
./HIP/tests/catch/unit/texture/hipTextureObjFetchVector.cc
./samples/texture/hipTextureObjFetchVector.cc

Seems like only this file is mirroring a unit test name. I can rename and
re-enable?

**Comment by linehill** [comment]
You can remove the ./samples/texture/hipTextureObjFetchVector.cc one. The test case was borrowed from the HIP-Common. I recall that the HIP-Common test was not added to the catch framework when I did the borrow it.

**Pull request merged**